server/package.json

```diff
@@ -15,19 +15,27 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@azure/identity": "^4.2.0",
+    "@azure/identity": "^4.4.1",
     "@azure/storage-blob": "^12.23.0",
     "@sendgrid/mail": "^8.1.3",
     "axios": "^1.7.2",
     "azure-storage": "^2.10.7",
     "body-parser": "^1.20.2",
     "cors": "^2.8.5",
     "dotenv": "^16.4.5",
+<<<<<<< Updated upstream
```

Comment:
These should not be appearing like this. Somewhere you didn't stash your changes properly and you pushed without resolving conflicts.